socorro/signature/siglists/irrelevant_signature_re.txt

```diff
@@ -18,6 +18,7 @@ app_process@0x
 AppleIntelHD3000GraphicsGLDriver@
 __chk_fail
 _chkstk
+compiler_builtins:: # Covers memcpy, memset, __udiv and others
```

Comment:
I'm pretty sure the way the code is written now, you need to put comments on their own line.

```suggestion
# Covers memcpy, memset, __udiv and others
compiler_builtins::
```